lib/ar_query_matchers.rb

```diff
@@ -7,6 +7,20 @@
 
 module ArQueryMatchers
   module ArQueryMatchers
+    class Utility
+      def self.clean_expected(expected)
+        expected_cleaned = expected.select { |k,v| v > 0 }
+
+        # On the fence about whether or not to include a warning message. Folks writing tests should be aware that they
+        #   should remove the 0 count expectations to keep the tests clean. Problem is it makes the output for this test
+        #   suite noisy. I could go either way on this
+        # amount_cleaned = expected.size - expected_cleaned.size
+        # puts "Removed #{amount_cleaned} 0 count expectation(s), please remove from your test to hide this message" if amount_cleaned > 0
```

Comment:
I like the idea of informing the engineer that their test has a superfluous zero count. However it makes the tests local to this gem noisy. Happy to go either way on this one. Obviously, I'd clean up this method if we choose to not notify, as this method will go down to a one-liner. If we do choose to notify, I couldn't use pluralize, given we're not in Rails land, and would appreciate a way to do the pluralization without rolling our own (not hard but duplicative).

Reply:
I can also see an argument for why you'd like to assert 0:
```ruby
expect { do_something }.to load_models('Employee' => 0, 'Company => 1)
```
vs

```ruby
expect { do_something }.to load_models('Company => 1)
```

The former allows you to convey the intent that you are explicitly expecting 0 employee queries.
In the latter it's implicit.